### PR TITLE
South-Africa-Rustumb-Marz

### PR DESCRIPTION
**Pull request by russmahn**
# Your Details

- Your Name:
- Your City:
- Your Slack Name:

# Homework Details

- Module:
- Week:

### CODE REVIEW

File: exercises/F-strings-methods/exercise.js [hunk 1]
```diff
@@ -1,3 +1,6 @@
 // Start by creating a variable `message`
-
-console.log(message);
+var name = "Rustumb";
+var nameLength = name.length;
+var myNameIs ="My name is ";
```
Comment (DouglasVDM) [inline exercises/F-strings-methods/exercise.js:4]: Good to see your use of naming yor variables for other users to be able to know what's going on in your program.

**Review by DouglasVDM — COMMENTED** [review]
Great job Rustumb,  with finishing the exercises.

File: exercises/G-numbers/exercise.js [hunk 2]
```diff
@@ -1 +1,6 @@
 // Start by creating a variables `numberOfStudents` and `numberOfMentors`
+var numberOfStudents = 15;
+var numberOfMentors = 8;
+var total = numberOfStudents + numberOfMentors;
+var message = 'Number of students: ' + numberOfStudents + '\nNumber of mentors: ' + numberOfMentors + '\nTotal number of students and mentors: ' + total;
```
Comment (sinopungula) [inline exercises/G-numbers/exercise.js:5]: Hey Rustumb! 

I am happy to see your efficiency in line 5. I made my code redundant by adding another variable for the number of mentors. Learnt something!

**Review by sinopungula — COMMENTED** [review]
Hey. Great work Rustumb!

**Pull request closed without merging**